code/index.html

```diff
@@ -4,19 +4,139 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header container">
+      <a href="#"><div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div></a>
```

Comment:
Nice with a hamburger! I think you can skip the `a` tag though. If you want it to look like it's clickable, you can add `cursor: pointer;` in CSS :)